.circleci/test-deploy.yml

```diff
@@ -16,6 +16,10 @@ executors:
   linux:
     docker:
       - image: cimg/base:stable
+  linux-arm:
+    docker:
+      - image: cimg/base:stable
+    resource_class: arm.medium
```

Comment:
The cimg/base image don't have an ARM alternative, so this won't work.
To have ARM executors, they should be machine type.